pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,417 @@
+         When output voltage is enabled green led is ON and the
+         voltage_setting will be present on the output""",
+        validator=strict_discrete_set,
+        values={True: 'ON', False: 'OFF'},
+        map_values=True
+    )
+
+    id = Instrument.measurement(
+        "ID",
+        """ Returns the identification of the instrument """,
+        get_process=lambda r:
+        r[1].strip().encode(EurotestHPP120256.response_encoding).decode('utf-8', 'ignore')
+    )
+
+    status = Instrument.measurement(
+        "STATUS,DI",
+        """ Returns the unit Status which is a 16bits response where
+        every bit indicates the state of one subsystem of the HV Source """,
+        # response DI, b15 b14 b13 b12 b11 b10 b9 b8 b7 b6 b5 b4 b3 b2 b1 b0,
+        #               0                   1
+        # IpErr  b15    no input error      input error
+        # Ramp   b14    no ramp             ramp
+        # CutOut b13    -                   emergency off
+        # TpErr  b12    no trip error       trip error
+        # F3     b11                reserved
+        # F2     b10                reserved
+        # menu1  b9     submenu off         submenu on
+        # menu0  b8     menu off            menu on
+        # err    b7     no error            error
+        # Creg   b6     no current control  current control
+        # Vreg   b5     no voltage control  voltage control
+        # pol    b4     negative            positive
+        # inh    b3     no ext. inhibit     external inhibit
+        # local  b2     remote              local
+        # kilena b1     kill disable        kill enable
+        # on     b0     off                 high voltage is ON
+        get_process=lambda r:
+        EurotestHPP120256.EurotestHPP120256Status(
+            int(r[1].strip()[:-1].encode(EurotestHPP120256.response_encoding).
+                decode('utf-8', 'ignore'), 2)
+        )
+
+    )
+
+    lam_status = Instrument.measurement(
+        "STATUS,LAM",
+        """ Returns the LAM status which is the status of the unit from the point
+        of view of the process. Fo example, as a response of asking STATUS,LAM, the HV
+        voltage could response one of the messages from the next list:
+        LAM,ERROR External Inhibit occurred during Kill enable
+        LAM,INHIBIT External Inhibit occurred
+        LAM,TRIP ERROR Software current trip occurred
+        LAM,INPUT ERROR Wrong command received
+        LAM,OK Status OK""",
+        get_process=lambda r:
+        r[1].strip().encode(EurotestHPP120256.response_encoding).decode('utf-8', 'ignore')
+    )
+
+    def emergency_off(self):
+        """ The output of the HV source will be switched OFF permanently and the values
+        of the voltage and current settings set to zero"""
+        log.info("Sending emergency off command to the instrument.")
+
+        self.write("EMCY OFF")
+
+    def shutdown(self, voltage_rate):
```

Comment:
Does it make sense to add a default value for the `voltage_rate` (e.g. the maximum of `VOLTAGE_RAMP_RANGE`), so that users only have to call `shutdown()` for it to safely and quickly shut down?

Reply:
Sure. Done in commit. Thanks.